src/assets/scss/transition/_components.scss

```diff
@@ -36,6 +36,7 @@
 @import '../../@koop-components/common/loader/loader';
 @import '../../@koop-components/common/to-top/to-top';
 @import '../../@koop-components/common/highlight/highlight';
+@import '../../@koop-components/common/copydata/copydata';
```

Comment:
Is dit ook nodig, als ie niet anders is dan in components?

Reply:
de transition build bouwt volledig zelfstandig. Dus je hebt ook alle includes daar nodig.